libr/core/cmd_print.c

```diff
@@ -5852,6 +5852,162 @@ l = use_blocksize;
 				}
 			}
 			break;
+		case 't': // "pst"
+			if (l > 0) {
+				/*SdbListIter *iter;
+				SdbKv *kv;
+				SdbList *sdbls = sdb_foreach_list (core->anal->sdb_meta, true);
+				ut8 out[len];
+				ut8 c;
+				int i = 0;
+
+				for (int j = 0; j < len; j ++) {
+					ls_foreach (sdbls, iter, kv) {
+							char hex_str[2];
+							memcpy(hex_str, sdbkv_key (kv)+2, sizeof(char)*2);
+							int hex = (int)strtol(hex_str, NULL, 16);
+							if (core->block[j] == hex) {
+									c = (ut8) sdbkv_value (kv)[0];
+									out[i] = c;
+									i++;
+							}
+							out[i] = '\0';
+						}
+					}*/
+
+				charset pokered[61] = {
+											{ '\0', 0x50 },//terminating byte
```

Comment:
omg my eyes. please move all this code into a separate function and fix the indentation